dev-docs/bidders/themoneytizer.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: bidder
+title: The Moneytizer
+description: Prebid The Moneytizer Bid Adapter
+biddercode: themoneytizer
+media_types: display
+gvl_id: 1265
+pbjs: true
+userIds: criteo, id5Id, pubCommonId, unifiedId, quantcastId, lotamePanoramaId, sharedId
```

Comment:
looks like your adapter is passing all eids, any reason this list is included?

Reply:
it's fixed